test/test_serialize_load_commands.rb

```diff
@@ -0,0 +1,245 @@
+require "minitest/autorun"
+require "helpers"
+require "macho"
+
+class MachOLoadCommandSerializationTest < Minitest::Test
+  include Helpers
+
+  def test_serialize_segment
+    pass
+  end
+
+  def test_serialize_symtab
+    pass
+  end
+
+  def test_serialize_symseg
+    pass
+  end
+
+  def test_serialize_thread
+    pass
+  end
+
+  def test_serialize_unixthread
+    pass
+  end
+
+  def test_serialize_loadfvmlib
+    pass
+  end
+
+  def test_serialize_ident
+    pass
+  end
+
+  def test_serialize_fvmfile
+    pass
+  end
+
+  def test_serialize_prepage
+    pass
+  end
+
+  def test_serialize_dysymtab
+    pass
+  end
+
+  def test_serialize_load_dylib
+    filenames = SINGLE_ARCHES.map { |a| fixture(a, "hello.bin") }
+
+    filenames.each do |filename|
+      file = MachO::MachOFile.new(filename)
+      ctx = MachO::LoadCommand::SerializationContext.context_for(file)
+      lc = file[:LC_LOAD_DYLIB].first
+      lc2 = MachO::LoadCommand.create(:LC_LOAD_DYLIB, lc.name.to_s,
+        lc.timestamp, lc.current_version, lc.compatibility_version)
+      blob = lc.view.raw_data[lc.view.offset, lc.cmdsize]
```

Comment:
See also my earlier code comment that might have went unnoticed and is now collapsed by default: https://github.com/Homebrew/ruby-macho/pull/38#discussion_r70019985